Gemfile

```diff
@@ -98,6 +98,8 @@ gem 'gon', '~> 4.0'
 # don't require by default, instead load on-demand when actually configured
 gem 'airbrake', '~> 4.1.0', require: false
 
+gem 'transactional_lock', git: 'https://github.com/finnlabs/transactional_lock.git', branch: 'master'
```

Comment:
Wouldn't that mean I am only installing it for those platforms?
What about the `jruby` platform below?

I don't quite see how this gem should be specific to a certain platform...